src/sound_check_menu.c

```diff
@@ -0,0 +1,2198 @@
+            {
+                PlaySE12WithPanning(gUnknown_020387B4[CRY_TEST_UNK0], -0x40);
+                gUnknown_020387B4[CRY_TEST_CHORUS] = -0x40;
+                gUnknown_020387B4[CRY_TEST_PROGRESS] = 1;
+                gUnknown_020387B4[CRY_TEST_RELEASE] = 0x1E;
+                return;
+            }
+        }
+        else // _080BB140
+        {
+            PlaySE12WithPanning(gUnknown_020387B4[CRY_TEST_UNK0], 0x3F);
+            gUnknown_020387B4[CRY_TEST_CHORUS] = 0x3F;
+            gUnknown_020387B4[CRY_TEST_PROGRESS] = 1;
+            gUnknown_020387B4[CRY_TEST_RELEASE] = 0x1E;
+            return;
+        }
+        // _080BB154
+        PlaySE12WithPanning(gUnknown_020387B4[CRY_TEST_UNK0], panpot);
+        gUnknown_020387B4[CRY_TEST_PROGRESS] = 0;
+        return;
+    }
+    if(gMain.newKeys & 0x200) // _080BB15E
+    {
+        gUnknown_020387B4[CRY_TEST_PANPOT]++;
+        if(gUnknown_020387B4[CRY_TEST_PANPOT] > 4)
+            gUnknown_020387B4[CRY_TEST_PANPOT] = 0;
+    }
+    if(gMain.newKeys & 0x100) // _080BB176
+    {
+        gUnknown_020387B4[CRY_TEST_PANPOT]--;
+        if(gUnknown_020387B4[CRY_TEST_PANPOT] < 0)
+            gUnknown_020387B4[CRY_TEST_PANPOT] = 4;
+    }
+    if(gMain.newAndRepeatedKeys & 0x10) // _080BB192
+    {
+        gUnknown_020387B4[CRY_TEST_UNK0]++;
+        if(gUnknown_020387B4[CRY_TEST_UNK0] > 0xF7)
+            gUnknown_020387B4[CRY_TEST_UNK0] = 0;
+    }
+    else if(gMain.newAndRepeatedKeys & 0x20) // _080BB1B0
+    {
+        gUnknown_020387B4[CRY_TEST_UNK0]--;
+        if(gUnknown_020387B4[CRY_TEST_UNK0] < 0)
+            gUnknown_020387B4[CRY_TEST_UNK0] = 0xF7;
+    }
+}
+
+void sub_80BB1D4(void)
+{
+    u8 lrStr[5];
+    u8 rlStr[5];
+
+    memcpy(lrStr, gOtherText_LR, sizeof lrStr);
+    memcpy(rlStr, gOtherText_RL, sizeof rlStr);
+
+    sub_80BAE78(gUnknown_020387B4[CRY_TEST_UNK0], 7, 2, 3);
+
+    switch(gUnknown_083D03F8[gUnknown_020387B4[CRY_TEST_PANPOT]])
+    {
+        case 0x7F:
+            MenuPrint(lrStr, 7, 4);
+            break;
+        case -0x80:
+            MenuPrint(rlStr, 7, 4);
+            break;
+        default:
+            sub_80BAE78(gUnknown_083D03F8[gUnknown_020387B4[CRY_TEST_PANPOT]], 7, 4, 3);
+            break;
+    }
+    sub_80BAE78(IsSEPlaying(), 12, 8, 1);
+}
+
+void sub_80BB25C(u8 taskId)
+{
+    struct CryRelatedStruct cryStruct, cryStruct2;
+    u8 zero;
+
+    SetUpWindowConfig(&gWindowConfig_81E6C3C);
+    InitMenuWindow(&gWindowConfig_81E6CE4);
+    gUnknown_03005D34 = 1;
+    ResetSpriteData();
+    FreeAllSpritePalettes();
+
+    cryStruct.unk0 = 0x2000;
+    cryStruct.unk2 = 29;
+    cryStruct.paletteNo = 12;
+    cryStruct.yPos = 30;
+    cryStruct.xPos = 4;
+
+    zero = 0; // wtf?
+    gUnknown_03005E98 = 0;
+
+    while(sub_8119E3C(&cryStruct, 3) == FALSE);
+
+    cryStruct2.unk0 = 0;
+    cryStruct2.unk2 = 15;
+    cryStruct2.paletteNo = 13;
+    cryStruct2.xPos = 12;
+    cryStruct2.yPos = 12;
+
+    zero = 0; // wtf?
```

Comment:
Personal opinion: non-explanatory comments don't really add anything to the codebase.